Components/Atoms/GoalListItem.js

```diff
@@ -253,21 +254,28 @@ const GoalListItem = ({
         duration: 100,
       }),
     ]).start(() => {
-      Animated.timing(moveAnim, {
-        toValue: 0,
-        duration: 0,
-      }).start(() => {
-        Animated.timing(fadeAnim, {
-          toValue: 1,
-          duration: 1500,
-        }).start();
+      pushGoal().then(() => {
+        Animated.timing(moveAnim, {
+          toValue: 0,
+          duration: 0,
+        }).start(() => {
+          Animated.timing(fadeAnim, {
+            toValue: 1,
+            duration: 1500,
```

Comment:
@ChrisCruze this is like 1.5 seconds right? Can we bring it down to like to 1/3 or 1/2 of a second? Also, I think just the fade animation is good enough. I think we should try it with no moveAnim. 

Animation seems to be slow on Expo/RN. Let's try and minimize it where we can?